File: src/exercicios/impostoSalario.py
def calcular_imposto(salario):
    imposto = 0.00
    if (salario >= 0 and salario <= 1100):
        imposto = 0.05
    elif (salario >= 1100.1 and salario <= 2500):
        imposto = 0.10
    else:
        imposto = 0.15
    return imposto * salario

salario = float(input("Qual o valor do seu salário?"))
beneficio = float(input("Qual o valor do seu beneficio?"))

imposto = calcular_imposto(salario)

saida = salario - imposto + beneficio

print("Seu salário após o desconto do imposto (R$ " + f'{imposto: .2f}' + ") é R$ " + f'{saida: .2f}')

File: src/exercicios/ImpostoSalario.java
package exercicios;
import java.util.Scanner;

public class ImpostoSalario{
    /**
     * @param salario - valor do salario dado em float
     * 
     */
    public static void main(String[] args) {

        Scanner vScanner = new Scanner(System.in);

        System.out.println("Qual o valor do seu salário? ");
        float salario = vScanner.nextFloat();

        System.out.println("Qual o valor dos seus benefícios? ");
        float beneficio = vScanner.nextFloat();
        
        float imposto = 0;
        if (salario >= 0 && salario <= 1100) {
            imposto = 0.05F * salario;
        } else if (salario >= 1100.1 && salario <= 2500) {
            imposto = 0.10F * salario;
        } else {
            imposto = 0.15F * salario;
        } 

        float saida = salario - imposto + beneficio;

        System.out.println("Seu salário após o desconto do imposto (R$ " + String.format("%.2f", imposto) + ") é R$ " + String.format("%.2f", saida));
    }

}